jsonmerge_utils.py

```diff
@@ -5,17 +5,58 @@
 @author: GRENTOR
 """
 import os
+import argparse
+import sys
 import logging
 #from tqdm import tqdm
 from merge_files import Merge
+
+def get_parser():
+    """
+    gets & parses arguments from command line
+    """
+    parser = argparse.ArgumentParser(
+        description="Merges different JSON files into a single JSON object in a new file",
+        add_help=True)
+    parser.add_argument('-ip',
+                        dest = "input_prefix",
+                        help="Input prefix",
+                        required=True)
+    parser.add_argument('-op',
+                        dest = "output_prefix",
+                        help="Output prefix",
+                        required=True)
+    parser.add_argument('-maxFileSize',
+                        dest = "max_file_size",
+                        help="The maximum file size (in bytes) that each merged file should have",
+                        type=int,
+                        required=True
+                        )
+    parser.add_argument('-dir_path',
+                        dest ="data_dir",
+                        help="Path to directory where all json files are stored",
+                        required=True)
+    parser.add_argument('-log_level',
+                        dest="log_level",
+                        help="The logging level - defaults to INFO",
+                        metavar = "{'INFO','DEBUG','ERROR'}",
+                        choices = ['INFO', 'DEBUG', 'ERROR'],
+                        default="INFO")
+    if len(sys.argv) == 1:
+        parser.print_help(sys.stderr)
+        sys.exit(1)
+    return parser.parse_args()
+
+
 logger = logging.getLogger(__name__)
-def configure_logger():
+def configure_logger(args):

```

Comment:
we can pass just the log_level argument here, passing the whole args object seems unnecessary

Reply:
```suggestion
def configure_logger(log_level):

```